src/Common/Exception.cpp

```diff
@@ -261,7 +261,7 @@ static void getNotEnoughMemoryMessage(std::string & msg)
             }
         }
 
-        if (num_maps > max_map_count * 0.99)
+        if (num_maps > max_map_count * 0.90)
```

Comment:
This change is not related, but it may be useful to debug failures like this: https://s3.amazonaws.com/clickhouse-test-reports/0/349f1b140a8d3a7c4c4b7c05fa8697f1ca73bd38/stateful_tests__debug__actions_.html